### PR TITLE
Add support for moonlander (and other potential templates)

### PR DESCRIPTION
**Pull request by vladzaharia**
- Added support for multiple keyboard layouts based on geometry
- Added support for Moonlander Mk 1

Todo:
I'd like to add support for Planck's keyboard layout soon

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,362 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (vladzaharia) [inline .gitignore:1]: This is just the standard C# gitignore from:

https://github.com/github/gitignore/blob/master/VisualStudio.gitignore

**Comment by vladzaharia** [comment]
Moonlander;
image

Planck:
image

**Comment by Invvard** [comment]
First of all, thank you so much for this, seeing someone contributing is so motivating !
I'll review your PR tomorrow (AFK for the day) but as long as your code is tested and the existing code is working, that should go smoothly :)

**Comment by vladzaharia** [comment]
It's an awesome tool, I just ordered my moonlander and thought it'd be perfect for it so happy to bring support for it. :)

Made sure to pass all the tests as well, of course:
image

File: src/InvvardDev.EZLayoutDisplay.Tests/Service/LayoutServiceTest.cs [hunk 2]
```diff
@@ -14,27 +14,29 @@ public class LayoutServiceTest
     {
```
Comment (vladzaharia) [inline src/InvvardDev.EZLayoutDisplay.Tests/Service/LayoutServiceTest.cs:14]: VS decided to get a little aggressive with the auto-formatting here, I'll revert it.

**Comment by vladzaharia** [comment]
Fixed em all up and ready to review when you have the time @Invvard

Also added support for their new domain name for configurations (configure.zsa.io)

**Comment by Invvard** [comment]
I've tried fixing the CI pipeline but failed... I've used legacy for quite sometime.
I'll fix it while putting it in YAML

**Pull request merged**